src/Infrastructure.VS/ThreadHandling.cs

```diff
@@ -31,6 +32,8 @@
 namespace SonarLint.VisualStudio.Infrastructure.VS
 {
     [ExcludeFromCodeCoverage] // Simple wrapper around hard-to-test VS types
+    [Export(typeof(IThreadHandling))]
```

Comment:
Core cannot create this class as it sits in Infrastructure.VS.
There is no reason not to mef-export it, so core can just mef-import it.